### PR TITLE
Update everything

### PR DESCRIPTION
**Pull request by cibernox**
This is the first stepping stone into refactoring this component to glimmer.

### CODE REVIEW

File: tests/integration/components/head-tag-test.js [hunk 1]
```diff
@@ -58,7 +58,6 @@ module('Integration | Component | head tag', function(hooks) {
       'src',
       'property',
       'itemprop',
-      'id'
```
Comment (cibernox) [inline tests/integration/components/head-tag-test.js:61]: This is removed because it makes no sense. Further down in the test this ID is being overriden.

**Comment by snewcomer** [comment]
We might need to add a github actions and rm travis.  I'm not sure why none of the tests ran for this PR.

https://github.com/offirgolan/ember-validators/pull/93/files

File: tests/integration/components/head-tag-test.js (hunk 1 above)
Comment (snewcomer) [reply tests/integration/components/head-tag-test.js:61]: https://github.com/ronco/ember-cli-meta-tags/pull/80

On interesting

Comment (cibernox) [reply tests/integration/components/head-tag-test.js:61]: I saw that, but I still don't get it. When we render the component on this very same test with `await render(hbs`{{head-tag headTag=headTag id='test-comp'}}`);` we are passing `id='test-comp'`, which precisely overwrites the ID in `headTag`.

What I don't understand is how this even worked to begin with.

**Comment by snewcomer** [comment]
@cibernox 👋 😃.  Looks like this has https://github.com/ronco/ember-cli-meta-tags/pull/105 in this PR.  Are we missing something? I see

> Cannot use 'in' operator to search for 'collectHeadTags' in service

In the test logs...

**Comment by cibernox** [comment]
@snewcomer 👋 
Not really, we should just rebase it. However I have no clue why one of the tests is still failing.

**Comment by snewcomer** [comment]
Possible we could fix this test by chance?

Screen Shot 2021-08-10 at 8 45 45 AM

**Pull request closed without merging**